core/templates/dev/head/pages/about/about-page.controller.js

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 The Oppia Authors. All Rights Reserved.
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
```

Comment:
This should be 2016 only. 2019 for new files.

Reply:
Done.

Reply:
done

Reply:
done